modules/import/csv_file.py

```diff
@@ -129,7 +130,7 @@ def __parse_file(self):
         quote = self.options['quote_character']
         values = []
 
-        with open(filename, 'rU') as infile:
+        with open(filename, newline='') as infile:
```

Comment:
Switch to use python 3.7 documentation recommendation and example.
> If csvfile is a file object, it should be opened with newline=''.
https://docs.python.org/3.7/library/csv.html